kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -10,9 +10,12 @@
       <UiToolbar
         :removeNavIcon="showAppNavView"
         type="clear"
-        textColor="black"
+        :textColor="$themeTokens.appBarText || $themeTokens.text"
```

Comment:
What the usage looks like in the plugin:

https://github.com/learningequality/kolibri-instant-schools-plugin/pull/209/commits/4f968b5707e8f82ffec818c64d1e4f167d84739c